src/Sylius/Bundle/UserBundle/Exception/UserNotFoundException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\UserBundle\Exception;
+
+use Webmozart\Assert\InvalidArgumentException;
+
+class UserNotFoundException extends InvalidArgumentException
```

Comment:
`InvalidArgumentException` from Webmozart\Assert is used to provide backward compatibility in case someone is catching this kind of exception.